src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -26,9 +42,9 @@ export const TodoFilter: React.FC = () => {
             data-cy="statusSelect"
             onChange={onChangeStatus}
           >
-            <option value="all">All</option>
-            <option value="active">Active</option>
-            <option value="completed">Completed</option>
+            <option value={Status.ALL}>All</option>
+            <option value={Status.ACTIVE}>Active</option>
+            <option value={Status.COMPLETED}>Completed</option>
```

Comment:
```suggestion
           {Object.values(Status).map(status => (
             <option value={status}>{status}</option>
           ))}
```
It can be optimized (just change enum values to be uppercased by default)